xcube/cli/verify.py

```diff
@@ -40,6 +40,7 @@ def verify(cube):
       variables, if any;
     * has any data variables and that they are valid, e.g. min. 3-D, all have
       same dimensions, have at least dimensions "time", "lat", "lon".
+    * spatial coordinates and their corresponding bounds (if exist) are equidistant
```

Comment:
Isn't it now:

```suggestion
    * spatial coordinates and their corresponding bounds (if exist) are equidistant
       and monotonically increasing or decreasing.
```

?